test/format/format.sh

```diff
@@ -123,6 +125,13 @@ while :; do
 	-R)
 		timing_stress_split_test=1
 		shift ;;
+        -r)
+		live_record_binary="$2"
+		if [ ! $(command -v "$live_record_binary") ]; then
+			echo "$name: -r option argument \"${live_record_binary}\" does not exist in path"
```

Comment:
Consider linking the wiki page in case someone tries to use UndoDB without setting it up first. https://wiki.corp.mongodb.com/display/KERNEL/UndoDB+Usage

Reply:
8264d21